nilearn/plotting/tests/test_displays.py

```diff
@@ -52,3 +52,5 @@ def test_contour_fillings_levels_in_add_contours():
     # In this case, we simply omit appending inf
     oslicer.add_contours(img, filled=True, colors='b',
                          alpha=0.1, levels=[0., 0.2])
+
+    oslicer.add_contours(img, filled=True, levels=[0., 0.2])
```

Comment:
I haven't. I am waiting to push with some other changes integrated which we are discussing on how to choose/set 'levels'. I will work on those comments and push them.